tests/by-util/test_cp.rs

```diff
@@ -1072,3 +1072,67 @@ fn test_cp_one_file_system() {
         }
     }
 }
+
+#[test]
+#[cfg(target_os = "linux")]
```

Comment:
unix instead of linux doesn't work here?

Reply:
It certainly does not work on macOS yet (I'm in the process of implementing it (https://github.com/uutils/coreutils/issues/1773)).  I do not know about other unixes. 

Even on Linux it works only if the underlying filesystem supports copy-on-write (e.g. btrfs) hence the minimal checks done in test_cp_reflink_always().